<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="device" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>501</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/icons/device.svg);
background-color: rgba(255, 255, 255, 0);</string>
     </property>
    </widget>
    <widget class="QLabel" name="blue">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>15</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: blue;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="green">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>15</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: green;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="red">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>15</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="power">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/power.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}

QPushButton:checked {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/pause.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>420</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/stop.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/up.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="okButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>370</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/ok.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/down.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="restart">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>370</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/start.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="menuButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
image: url(:/icons/menu.svg);
background-color: rgb(222, 221, 218);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="powerOFF" native="true">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>104</y>
       <width>242</width>
       <height>196</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color: rgb(0, 0, 0);
}</string>
     </property>
     <widget class="QWidget" name="topBar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>242</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(246, 245, 244);
}</string>
      </property>
      <widget class="QProgressBar" name="batteryProgress">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>1</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {border: 1px solid black;border-radius: 5px;background-color: transparent;}
QProgressBar::chunk {background-color: rgb(46, 194, 126);}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="listWindow" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>242</width>
        <height>176</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>130</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QTextEdit" name="treatmentTimer">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{
	border: none
}
font: 22pt &quot;DejaVu Sans&quot;;</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QListWidget" name="menuList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>242</width>
         <height>185</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;DejaVu Sans&quot;;</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="sessionEnd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>181</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit{
	border: none
}</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; font-size:14pt;&quot;&gt;SESSION ENDS&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans'; font-size:14pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; font-size:14pt;&quot;&gt;Please go back &lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; font-size:14pt;&quot;&gt;to main menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="setTimePage" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>151</height>
        </rect>
       </property>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>194</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Please set the date and time:</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border:none;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans';&quot;&gt;Press menu button when finished&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QListWidget" name="historyList">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>242</width>
         <height>178</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;DejaVu Sans&quot;;</string>
       </property>
      </widget>
      <zorder>setTimePage</zorder>
      <zorder>progressBar</zorder>
      <zorder>sessionEnd</zorder>
      <zorder>menuList</zorder>
      <zorder>historyList</zorder>
      <zorder>treatmentTimer</zorder>
     </widget>
    </widget>
    <widget class="QCheckBox" name="batteryIndicator">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>430</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
min-width: 40px;
min-height: 40px;
 }

QCheckBox::indicator:checked
  {
    image: url(:/icons/battery.svg);
  }
 QCheckBox::indicator:unchecked
  {
    image: url(:/icons/low_power.svg);
  }</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>831</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(246, 245, 244);</string>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="backEnd">
     <attribute name="title">
      <string>Backend</string>
     </attribute>
     <widget class="QFrame" name="measurementFrame_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="attach">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>8</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox::indicator {
min-width: 100px;
min-height: 30px;
 }

QCheckBox::indicator:checked
  {
    image: url(:/icons/true_.svg);
  }
 QCheckBox::indicator:unchecked
  {
    image: url(:/icons/false.svg);
  }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="attachLabel">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>8</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt;</string>
       </property>
       <property name="text">
        <string>CONTACT</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="progressFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QSlider" name="siteSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>321</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="currentSiteTextLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>81</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Current site:</string>
       </property>
      </widget>
      <widget class="QLabel" name="siteLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>21</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="measurementFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="maxFreq">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="maxFreqAmp">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>62</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="maxFreqTextLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Max. frequency:</string>
       </property>
      </widget>
      <widget class="QLabel" name="maxFreqAmpTextLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Max. amplitude:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="freq" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>391</width>
        <height>191</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0)</string>
      </property>
     </widget>
     <widget class="QWidget" name="dft" native="true">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>260</y>
        <width>391</width>
        <height>191</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0)</string>
      </property>
     </widget>
     <widget class="QFrame" name="batteryChargeFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>341</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="chargeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Charge/discharge to:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="chargeSpinBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QPushButton" name="updateBatteryButton">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Update battery</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="PCend">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <attribute name="title">
      <string>PC end</string>
     </attribute>
     <widget class="QGraphicsView" name="PCList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>831</width>
        <height>471</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/icons/pc.svg);
background-color: rgba(255, 255, 255, 0);

</string>
      </property>
     </widget>
     <widget class="QListWidget" name="PCrecord">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>731</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget::item {
    color:rgb(0, 0, 0);
    background-color:rgb(249, 249, 249);
	selection-background-color: rgb(38, 162, 105);
}
QScrollBar:vertical {              
            background:rgb(249, 249, 249);
 }
QListWidget{
	background-color:rgb(249, 249, 249);
	font: 8pt &quot;DejaVu Sans&quot;;
}</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>tab</zorder>
   <zorder>device</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
